### ISSUE TITLE
Incorrectly fails on 2 or more consecutive template literals

### ISSUE THREAD

**Issue by TimeRaider** [styled-components/stylelint-processor-styled-components#20]
```js
import styled from 'styled-components';
const gap = '5px';
const Button = styled.div`
    padding: 0 ${gap} ${gap};
`
```
produces
`Missed semicolon   CssSyntaxError`

**Comment by nathggns** [comment]
+1

**Comment by ismay** [comment]
Closed in #52
